test/test_models.py

```diff
@@ -671,33 +688,46 @@ def test_vitc_models(model_fn, dev):
 def test_classification_model(model_fn, dev):
     set_rng_seed(0)
     defaults = {
-        "num_classes": 50,
+        "num_classes": 1000,
         "input_shape": (1, 3, 224, 224),
+        "num_expected": 50,
```

Comment:
Could you explain why this is needed?

If it is, we should make sure to find a descriptive name like `num_classes_to_check`

Reply:
This is needed to make sure our [expected](https://github.com/pytorch/vision/tree/main/test/expect) file small in size, otherwise when we use real_weight it will force the number of classes to be 1000 and our expected file will be 20x bigger than before.

I agree, I will change the name to `num_classes_to_check`